services/modrinth/modrinth-version.service.js

```diff
@@ -0,0 +1,28 @@
+import { renderVersionBadge } from '../version.js'
+import { BaseModrinthService, documentation } from './modrinth-base.js'
+
+export default class ModrinthVersion extends BaseModrinthService {
+  static category = 'version'
+
+  static route = {
+    base: 'modrinth/version',
```

Comment:
Normally we use `/v` as the route for version badges - see https://github.com/badges/shields/blob/master/doc/badge-urls.md